### PR TITLE
update nem2-sdk 0.16.2

### PR DESCRIPTION
**Pull request by dl-nice**

### CODE REVIEW

File: src/core/model/Listeners.ts [hunk 1]
```diff
@@ -71,6 +71,7 @@ export class Listeners {
         })
 
         this.listener.status(address).subscribe((error: TransactionStatusError) => {
+            // @ts-ignore
           Notice.trigger(error.status.split('_').join(' '), NoticeType.error, store)
```
Comment (decentraliser) [inline src/core/model/Listeners.ts]: error.code.split

Comment (dl-nice) [reply src/core/model/Listeners.ts]: Replace status with code?

Comment (dl-nice) [reply src/core/model/Listeners.ts]: ok

**Comment by decentraliser** [comment]
U gotta fix the error mock in listeners.spec as well

**Comment by dl-nice** [comment]
ok fix

File: __tests__/core/model/Listeners.spec.ts [hunk 2]
```diff
@@ -82,7 +82,7 @@ jest.mock('nem2-sdk/dist/src/infrastructure/Listener', () => ({
     close: mockListenerClose,
     confirmed: mockConfirmed,
     newBlock: mockNewBlock,
-    status: mockStatus,
+    code: mockStatus,
```
Comment (decentraliser) [inline __tests__/core/model/Listeners.spec.ts]: No

Comment (dl-nice) [reply __tests__/core/model/Listeners.spec.ts]: fix

**Pull request merged**